### DISCUSSION TITLE
Can reach Django admin page, can't reach Joplin

### DISCUSSION THREAD

**Discussion by senior-discount** [joplin-vieweb/joplin-vieweb#38] [Q&A] [answered]
Thank you for your work!
I am able to reach the Django admin page just fine (joplin.mysite.com/admin) but I can't reach joplin for some reason (I get a NOT FOUND error page). my joplin only yaml file is the same as yours however I only changed the web address in ORIGINS to mine as well as changed "<PORT-TO-YOUR-PROXY>" to the port I wanted Joplin to be exposed to. I have nginx instance in another VM running. I did not want to create and github issue of this because I'm sure I may just have configured something wrong. Any help is appreciate it. Thank you.

**Comment by gri38** [comment, marked as answer]
Hello.

Just to be sure: the expected url for joplin content is: joplin.mysite.com/**joplin** 
Do you input this url ?

If so, could you share your docker-compose file and your nginx config file (the front nginx: yours, in the separate VM).

You also can enable debug mode of django (I'm on vacancion, without access to my computer, so I write from my memory...): edit the settings file:
/var/lib/docker/volumes/XXX_joplin-vieweb/_data/settings.py
and change
```python
DEBUG = False
```
to 
```python
DEBUG = True
```

Restart the docker-compose.
And see if you have more info on the web page.
You also can have a look to the logs of django
```bash
sudo docker logs --follow ID_OF_CONTAINER_running gri38/django-joplin-vieweb
```
and see what happen when you try to access joplin.

to be continued ;-)

**Reply by senior-discount** [reply]
I don't know how I missed that the url to access joplin was joplin.mysite.com/joplin. Could you specify that install instructions that admin page is <you-url>/admin and that joplin is access through <your-url>/joplin? Thanks again.

**Reply by gri38** [reply]
Hi.
Happy this was "just" that. I've read the joplin-vieweb readme and indeed the usage urlis "hidden" in the users configuration paragraph. I'll update it just after installation instructions I'll write the urls for admin and usage.

You've paid the price of the doc maturity... thx ;-)
